__tests__/DateBlock.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { DateBlock } from '../components';
+
+describe('Component: DateBlock', () => {
+  const props = {};
+
+  beforeEach(() => {
+    props.date = '2015-03-14T12:00:00Z';
+  });
+
+  it('renders without crashing', () => {
+    const wrapper = shallow(<DateBlock {...props} />);
+    expect(wrapper).toHaveLength(1);
```

Comment:
yeah, i'm getting used to a different paradigm with nest myself.

jest snapshots let you avoid asserting against selectors. you just assert it matches the snapshot, and the first time the test run it creates a snapshot. so instead of `.toHaveLength(1)` you just assert `.toMatchSnapshot()`

that sounded awful to me at first, but it really simplifies view testing, and when you make a pr with a snapshot diff, it has the benefit of documenting how the rendered dom changes.